configure.ac

```diff
@@ -106,6 +106,37 @@ AC_MSG_CHECKING(
     [for vma extra api])
 AC_MSG_RESULT([${have_vma_api}])
 
+##########################################################################
+# check XLIO extra API
+#
+AC_ARG_ENABLE(
+    [xlio-api],
+    AC_HELP_STRING([--enable-xlio-api],
+                   [SOCKPERF: enable xlio extra api support: 'yes', 'no' or library installation path (default=no)]),
+    [have_xlio_api=$enableval],
+    [have_xlio_api=no])
+AS_IF([test "${have_xlio_api}" != "no"],
+    [
+    if test "$have_xlio_api" = "yes"
+    then
+        have_xlio_api=/usr
+    fi
+    CPPFLAGS="$CPPFLAGS -I$have_xlio_api/include"
+    if test -d "$have_xlio_api/lib64"
+    then
+        LDFLAGS="$LDFLAGS -L$have_xlio_api/lib64"
+    else
+        LDFLAGS="$LDFLAGS -L$have_xlio_api/lib"
+    fi
```

Comment:
no need to set LDFLAGS. See https://github.com/Mellanox/sockperf/pull/188#discussion_r910731572

Reply:
So we do need CPPFLAGS but dont need LDFLAGS?

Reply:
I prefer to return to https://github.com/Mellanox/sockperf/blame/660800577bef42036ce090fbabb217b7df1efbc2/configure.ac
as more flexible solution. Only thing you need is just point correct path in CPPFLAGS="-I<path to vma> -I<path to xlio>"

Reply:
We don't need to alter LDFLAGS because we're not directly linking to vma or xlio. We use only a functions defined in header file.

I disagree with reverting to the old solution because it's useful to specify alternative path to headers.

Reply:
> I disagree with reverting to the old solution because it's useful to specify alternative path to headers.
Adding one line as `CPPFLAGS="$CPPFLAGS -I/usr/include"` to previous solution allows to get alternative path.

Reply:
In this case you need to edit configure.ac instead of passing `--enable-xlio-api=/alternative/path/to/xlio/installation` in command line

Reply:
> In this case you need to edit configure.ac instead of passing `--enable-xlio-api=/alternative/path/to/xlio/installation` in command line

Sorry I missed
```
    if test "$have_xlio_api" = "yes"
    then
        have_xlio_api=/usr
    fi
```

Reply:
Done